.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Thanks for sending a pull request!
+
+Please make sure you've read and understood our contributing guidelines
+(https://github.com/containers/buildah/blob/master/CONTRIBUTING.md) as well as ensuring
+that all your commits are signed with `git commit -s`.
```

Comment:
Is there anything to make this automatic?

Reply:
I'm not sure, need to dig further.  It certainly would be nice if we could pull that off.

Reply:
Hm no I don't think we could automate this without having a write access token to the repository.